tests/compute/loop-unroll.slang

```diff
@@ -4,7 +4,7 @@
 //TEST(compute):COMPARE_COMPUTE:-cpu -shaderobj
 //TEST(compute):COMPARE_COMPUTE:-cuda -shaderobj
 // Note VK output is not loop unrolled
-//TEST(compute):COMPARE_COMPUTE:-vk -shaderobj
+//DISABLE_TEST(compute):COMPARE_COMPUTE:-vk -shaderobj
```

Comment:
This will also disable the test for glslang path. Can we figure out a way to just disable the spirv path?

Reply:
yeah, there are a few oddities like this at the moment. I suggest disregard them for now until this is in a mergeable state.